digits/scheduler.py

```diff
@@ -175,6 +177,30 @@ def add_job(self, job):
             return False
         else:
             self.jobs.append(job)
+
+            if flask._app_ctx_stack.top is not None:
```

Comment:
That looks like it could be some sort of circular import issue. Are you messing with any import statements in scheduler or webapp? The import order is very touchy and delicate.